openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -485,8 +485,8 @@ If a scope value related to credential issuance and the `authorization_details`
 
 This specification defines the following request parameters that can be supplied in an Authorization Request:
 
-* `wallet_issuer`: OPTIONAL. JSON String containing the Wallet's identifier. The Credential Issuer can use the discovery process defined in [@!SIOPv2] to determine the Wallet's capabilities and endpoints, using `wallet_issuer` value as an Issuer Identifier referred to in [@!SIOPv2]. This is RECOMMENDED in Dynamic Credential Requests.
-* `user_hint`: OPTIONAL. JSON String containing an opaque user hint the Wallet MAY use in subsequent callbacks to optimize the user's experience. This is RECOMMENDED in Dynamic Credential Requests.
+* `wallet_issuer`: OPTIONAL. String containing the Wallet's identifier. The Credential Issuer can use the discovery process defined in [@!SIOPv2] to determine the Wallet's capabilities and endpoints, using `wallet_issuer` value as an Issuer Identifier referred to in [@!SIOPv2]. This is RECOMMENDED in Dynamic Credential Requests.
+* `user_hint`: OPTIONAL. String containing an opaque user hint the Wallet MAY use in subsequent callbacks to optimize the user's experience. This is RECOMMENDED in Dynamic Credential Requests.
```

Comment:
There are  a lot of End-User (defined term) and user in the current text

would the term user be changed to End-User?

Please note: not of in the scope of this Pr, if of interest we can move this in an issue

Reply:
I think End-User makes sense. Can you please file an issue for this?

Reply:
thx, https://github.com/openid/OpenID4VCI/issues/104